core/gesture.ts

```diff
@@ -424,6 +407,12 @@ export class Gesture {
     this.hasStarted_ = true;
 
     blockAnimations.disconnectUiStop();
+
```

Comment:
Do this check before setting `this.hasStarted_` or calling `disconnectUiStop`.

Reply:
Originally had here so that it wouldn't be a change in behavior. But yeah, probably better to throw the error before we mess up the state of the gesture.